amgut/handlers/auth_handlers.py

```diff
@@ -44,8 +57,8 @@ def post(self):
             self.redirect(media_locale['SITEBASE'] + '/db_error/?err=regkit')
```

Comment:
the regkit value is actually a code for what error to show on the page, so it is text-localed.